modules/system/classes/MediaLibrary.php

```diff
@@ -535,6 +535,10 @@ public function getPathUrl($path)
     {
         $path = $this->validatePath($path);
 
+        if (Config::get('cms.storage.media.disk') === 'local') {
+            return $this->storagePath . $path;
+        }
+
         $fullPath = $this->storagePath.implode("/", array_map("rawurlencode", explode("/", $path)));
```

Comment:
Shouldn't you put the check after this line and return fullPath instead?

Reply:
Also @bennothommo can you explain why the code sniffer is saying that inline code structure isn't allowed? It shows up on line 742 in the filterItemList() method but this PR hasn't changed those lines.

Reply:
@LukeTowers I can't limit the code sniffer to specific lines unfortunately, so it checks the whole file. I figured this would be fine as it would encourage a gradual cleanup of the code.

Reply:
Oops, @Samuell1 wrong PR for my comment about fixing this. @panakour can you fix that line?

Reply:
@LukeTowers which line? I did not understand what is the problem

Reply:
@panakour This line here:
https://github.com/octobercms/october/blob/e82a3698efef756cf8efae2b02f5352f3ebdc088/modules/system/classes/MediaLibrary.php#L742
You'll need to add the curly brackets around the `return` call in order to pass the code quality checks.

Reply:
@bennothommo done

Reply:
@panakour Thanks for that. Also, did you have any thoughts as to @LukeTowers's question here? https://github.com/octobercms/october/pull/4423#discussion_r298671428

Reply:
I think it's better like i did it. It's unnecessary to execute the next line `        $fullPath = $this->storagePath.implode("/", array_map("rawurlencode", explode("/", $path)));
` if the storage disk is local.

Reply:
@panakour it's not unnecessary, that line exists for a reason - to URL encode any filenames that might have spaces or other illegal characters in them. Please make my suggested change so that the URL encoding happens for local files as well as remote files.